lib/msal-browser/src/interaction_handler/PopupHandler.ts

```diff
@@ -152,4 +152,15 @@ export class PopupHandler extends InteractionHandler {
             throw BrowserAuthError.createPopupWindowError(e.toString());
         }
     }
+
+    private cleanPopup(popupWindow?: Window): void {
+        if (popupWindow) {
+            // Close window.
+            popupWindow.close();
+        }
+        // Remove window unload function
+        window.onbeforeunload = null;
```

Comment:
This will also remove any event listeners added by customers. You should use `window.removeEventListener('unload', callback);`